proto/dydxprotocol/indexer/events/events.proto

```diff
@@ -184,6 +184,9 @@ message DeleveragingEventV1 {
   uint64 price = 5;
   // `true` if liquidating a short position, `false` otherwise.
   bool is_buy = 6;
+  // `true` if the deleveraging event is for final settlement, indicating
+  // the match occurred at the oracle price rather than bankruptcy price.
```

Comment:
Not: also note it's the bankruptcy price of the deleveraged subaccount (not offsetting)